systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -540,19 +541,26 @@
     
     % v gradient: dJdot_times_vi/dv
     [dtwistdv, v_indices] = geometricJacobian(model, kinsol, world, i, world);
-    %     dtwistdv = zeros(6, nv);
-    %     dtwistdv(:, v_indices) = J;
     nv_branch = length(v_indices);
     body_branch_velocity_num = nv_branch - length(body.velocity_num) + 1 : nv_branch;
     djoint_twistdv = zeros(6, nv_branch);
+    if isa(kinsol.J{i}, 'msspoly')
```

Comment:
can we change these into something like
```
djoint_twistdv = zeros(6, nv_branch) * J{i}(1)
```
It's ugly, but using isa is very slow.